src/overlays/actors/ovl_En_Heishi2/z_en_heishi2.c

```diff
@@ -50,11 +51,7 @@ void func_80A546DC(EnHeishi2* this, GlobalContext* globalCtx);
 void func_80A541FC(EnHeishi2* this, GlobalContext* globalCtx);
 void func_80A53DF8(EnHeishi2* this, GlobalContext* globalCtx);
 
-extern AnimationHeader D_06005C30;
-extern AnimationHeader D_06005500;
-extern SkeletonHeader D_0600BAC8;
-extern Gfx D_0602B060[]; // Keaton Mask
-extern Gfx D_06002C10[]; // 2D Guard in Window
+extern Gfx D_0602B060[]; // Keaton Mask From Object_Link_Child
```

Comment:
I think this should no longer be needed since object_link_child is included in the PR?